docs/sources/setup-grafana/configure-security/configure-authentication/google/index.md

```diff
@@ -68,3 +68,13 @@ When a user logs in using an OAuth provider, Grafana verifies that the access to
 Grafana uses a refresh token to obtain a new access token without requiring the user to log in again. If a refresh token doesn't exist, Grafana logs the user out of the system after the access token has expired.
 
 By default, Grafana includes the `access_type=offline` parameter in the authorization request to request a refresh token.
+
+## Skip organization role sync
+
+We do not currently sync roles from Google and instead set the AutoAssigned role to the user at first login. To be able to manage your users organization role from within Grafana , set `skip_org_role_sync` to `true`.
```

Comment:
@kalleep should we let them know that we are not syncronizating the roles. Than we should not mention it in the docs. 
I think we should mention it, as it is the truth and they could read it from the code anyways. 🤔 

Wdyt?